libs/model/src/policies/ContestWorker.policy.ts

```diff
@@ -35,21 +35,21 @@ export function ContestWorker(): Policy<typeof inputs> {
         const fullContentUrl = getThreadUrl({
           chain: community!.id!,
           id: payload.id!,
-          title: payload.title,
         });
         // content url only contains path
         const contentUrl = new URL(fullContentUrl).pathname;
 
-        const contestAddress = community!.contest_managers![0].contest_address;
-        const { address: userAddress } = (await models.Address.findByPk(
-          payload!.address_id,
-        ))!;
-        await contestHelper.addContent(
-          chainNodeUrl!,
-          contestAddress!,
-          userAddress,
-          contentUrl,
-        );
+        for (const contestManager of community!.contest_managers!) {
+          const { address: userAddress } = (await models.Address.findByPk(
+            payload!.address_id,
+          ))!;
+          await contestHelper.addContent(
```

Comment:
we should only add content if the thread belongs to the selected topics of each active contest